sphinx/application.py

```diff
@@ -796,14 +796,20 @@ def add_javascript(self, filename):
             StandaloneHTMLBuilder.script_files.append(
                 posixpath.join('_static', filename))
 
-    def add_stylesheet(self, filename):
+    def add_stylesheet(self, filename, alternate=None, title=None):
         self.debug('[app] adding stylesheet: %r', filename)
         from sphinx.builders.html import StandaloneHTMLBuilder
+        props = {}
+        if alternate is not None:
+            props['alternate'] = bool(alternate)
+        if title is not None:
+            props['title'] = title
```

Comment:
If I do that, and one doesn't pass a title, would e.g. ``{% if css_props[cssfile].title is defined %}`` work in the template? From a python point of view, it is defined (as ``None``).